OrbitLogic/Dockerfile

```diff
@@ -73,7 +73,7 @@ FROM base
 RUN apt update && apt-get install -y \
```

Comment:
Mixed use of `apt` and `apt-get`. This should be unified to use one or the other.